src/partials/messenger.ts

```diff
@@ -1267,6 +1267,21 @@ class NavigationController {
         this.searchVisible = !this.searchVisible;
     }
 
+    /**
+     * Toggle search bar off.
+     */
+    public toggleSearchOff(): void {
+        if (this.searchVisible) this.searchVisible = false;
+    }
+
+    /**
+     * Clear search when pressing ESC.
+     */
+    public clearSearch(ev): void {
+        if (ev.keyCode === 27) this.searchText = '';
```

Comment:
The `keyCode` field is deprecated and should not be used anymore: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode Instead, use `ev.key === 'Escape'`.

```suggestion
        if (ev.key === 'Escape') {
            this.searchText = '';
        }
```